react-frontend/src/pages/Home.js

```diff
@@ -46,30 +46,22 @@ class Home extends Component {
         <div className="container mt-5">
           <div className="row">
             <div className="col-lg-4">
-              <Sticker backgroundColor="#fe5e21">
-                <h1>Danger Zone</h1>
-              </Sticker>
+              <Sticker className="d-flex justify-content-center align-items-center mw-100 mh-100" backgroundColor="#fe5e21"><h1>Danger Zone</h1></Sticker>
             </div>
             <div className="col-lg-4">
-              <Sticker backgroundColor="#ffb30c">
-                <h1>Warning Zone</h1>
-              </Sticker>
+              <Sticker className="d-flex justify-content-center align-items-center mw-100 mh-100" backgroundColor="#ffb30c"><h1>Warning Zone</h1></Sticker>
             </div>
             <div className="col-lg-4">
-              <Sticker backgroundColor="#2ed351">
-                <h1>Safe Zone</h1>
-              </Sticker>
+              <Sticker className="d-flex justify-content-center align-items-center mw-100 mh-100" backgroundColor="#2ed351"><h1>Safe Zone</h1></Sticker>
             </div>
           </div>
-          <hr />
-          <center>
-            <p>
-              <strong>Made by Vinit Shahdeo</strong>
-            </p>
-          </center>
+          <hr/>
+          <p>
+            <center><strong>Made by Vinit Shahdeo</strong></center>
```

Comment:
@ashishnagpal2498 I dont encourage you to use `<center>` tag.

`<center>` is deprecated now. Please play around css or use [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/).